stdlib/public/core/StringStorage.swift

```diff
@@ -681,6 +681,8 @@ final internal class __SharedStringStorage
 
   internal var _breadcrumbs: _StringBreadcrumbs? = nil
 
+  internal var immortal = false
```

Comment:
At one point in time we were worried about the heap size of these objects. Does this change it? If so, would it make sense to put in `_countAndFlags`?

Reply:
I didn't consider changing `_countAndFlags`, that is a good thought. As is, 1 byte is added to the class's stored properties, so it goes from 32 to 33 bytes on 64-bit platforms.